### ISSUE TITLE
AssertionError: Invalid type <class 'NoneType'> for key WEIGHT_DECAY_BIAS

### ISSUE THREAD

**Issue by marifdemirtas** [facebookresearch/detectron2#3649]
Hi,
WEIGHT_DECAY_BIAS parameter in [`detectron2/config/defaults.py`](https://github.com/facebookresearch/detectron2/blob/main/detectron2/config/defaults.py) that exists for Detectron v1 training causes the mentioned error when you try to import the file. If you change the None value with 0 (line 560), the bug seems to be fixed but I am not sure if that change breaks something else.

## Instructions To Reproduce the 🐛 Bug:
1. In a Python terminal:
```from detectron2.config import configurable```
```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/storage/miniconda/envs/myenv/lib/python3.9/site-packages/detectron2/config/__init__.py", line 2, in <module>
    from .compat import downgrade_config, upgrade_config
  File "/storage/miniconda/envs/myenv/lib/python3.9/site-packages/detectron2/config/compat.py", line 28, in <module>
    from .defaults import _C
  File "/storage/miniconda/envs/myenv/lib/python3.9/site-packages/detectron2/config/defaults.py", line 560, in <module>
    _C.SOLVER.WEIGHT_DECAY_BIAS = None  # None means following WEIGHT_DECAY
  File "/storage/miniconda/envs/myenv/lib/python3.9/site-packages/fvcore/common/config.py", line 148, in __setattr__
    super().__setattr__(name, val)
  File "/storage/miniconda/envs/myenv/lib/python3.9/site-packages/yacs/config.py", line 155, in __setattr__
    _assert_with_logging(
  File "/storage/miniconda/envs/myenv/lib/python3.9/site-packages/yacs/config.py", line 521, in _assert_with_logging
    assert cond, msg
AssertionError: Invalid type <class 'NoneType'> for key WEIGHT_DECAY_BIAS; valid types = {<class 'float'>, <class 'list'>, <class 'str'>, <class 'int'>, <class 'bool'>, <class 'tuple'>}
```

## Environment:
I am using torch 1.10.0 with prebuilt detectron2 0.6 with CUDA 11.1. The mentioned codepiece still exists in the main branch, so I don't think this is related with my environment.

**Comment by hiyyg** [comment]
Hi @marifdemirtas , I met the same bug, have you solved it?

**Comment by hiyyg** [comment]
Hey, I solved it by updating yacs to >= 0.1.8.

**Comment by marifdemirtas** [comment]
Using the latest release of yacs (0.1.8) did not solve the problem for me. At the moment, I changed the None value in the defaults.py to 0 to bypass the issue.

**Comment by ppwwyyxx** [comment]
This is an environment issue because the import works in official colab tutorial. I believe it's because we require yacs>=0.1.8. 

If you need more help, please provide a way so that others can reproduce the issue.

**Comment by zhao-yiqun** [comment]
> Hey, I solved it by updating yacs to >= 0.1.8.

Hi, Following this method, I have updated yacs to 0.1.8,

However, when I run the code

from detectron2.config import get_cfg

there are still assertion error
AssertionError: Invalid type <class 'NoneType'> for key WEIGHT_DECAY_BIAS; valid types = {<class 'list'>, <class 'tuple'>, <class 'int'>, <class 'bool'>, <class 'str'>, <class 'float'>}

As I used the the conda to monitor my environments, when I use the conda list to check the version of yacs, it's version is 0.1.8
However, when I use the anaconda search -t conda yacs, it is still 0.1.6. Have you ever meet the same issue? many thanks to you~

**Comment by submagr** [comment]
@zhao-yiqun try following in your environment:
```
pip uninstall yacs
pip install yacs --upgrade
```

For some reason, upgrading yacs alone was not sufficient for me.
